src/scripts/sync_synonym.py

```diff
@@ -18,18 +18,23 @@
 from src.scripts.utils import PREFIX_MAP, get_owned_prefix_map
 
 
-# todo: when dirty_df no longer necessary, remove 'case' from these
+# todo: when combined_cases_df no longer necessary, remove 'case'
 HEADERS_TO_ROBOT_SUBHEADERS = {
     'mondo_id': 'ID',
     'mondo_label': '',
-    'synonym_scope': 'AI',  # todo next impt: pred  - 'A' correct? AI?
-    'synonym': '',  # todo: obj of 'scope' ow?
-    'source_id': '>A oboInOwl:source',  # todo axiom  # todo question: This is how we annotate source CURIEs. but isn't a CURIE similar to an IRI, so why isn't it 'AI' insead of just 'A'?
+    'synonym_scope': '',
+    'synonym_scope_mondo': '',
+    'synonym': '',
+    'synonym_exact': 'A oboInOwl:hasExactSynonym',
+    'synonym_broad': 'A oboInOwl:hasBroadSynonym',
+    'synonym_narrow': 'A oboInOwl:hasNarrowSynonym',
+    'synonym_related': 'A oboInOwl:hasRelatedSynonym',
```

Comment:
Why are these synonym scope columns added?

Reply:
It appears this is the only way that we can structure the ROBOT template (context: [1](https://docs.google.com/spreadsheets/d/1ftEZ47-JDe-1Z_fRWyO5NS39qT_-8MZsHXHEKz9aZ9Q/edit?gid=351291901#gid=351291901), [2](https://monarchinitiative.slack.com/archives/G01JACCEDAB/p1722628972648639?thread_ts=1722553603.356279&cid=G01JACCEDAB)).

I've left the original `synonym_scope` and `synonym` columns because I think these are better for humans / curation. But for ROBOT / machine, these 4 columns is what it needs.

Reply:
I think Trish is up to speed on these new columns so I am just doing some house cleaning and marking this resolved now.